omr/importpdf.cpp

```diff
@@ -165,64 +166,92 @@ void OmrState::importPdfMeasure(OmrMeasure* m, const OmrSystem* omrSystem)
 
 int OmrState::importPdfSystem(OmrSystem* omrSystem)
       {
-      for (int i = 0; i < omrSystem->measures().size(); ++i) {
-            OmrMeasure* m = &omrSystem->measures()[i];
-            importPdfMeasure(m, omrSystem);
-            }
-      LayoutBreak* b = new LayoutBreak(score);
-      b->setLayoutBreakType(LayoutBreak::Type::LINE);
-      score->lastMeasure()->add(b);
-      return tick;
+          for (int i = 0; i < omrSystem->measures().size(); ++i) {
```

Comment:
The indentation is messed up. See https://musescore.org/en/developers-handbook/musescore-coding-rules.
My Xcode settings
<img width="554" alt="capture d ecran 2016-05-25 11 10 24" src="https://cloud.githubusercontent.com/assets/234271/15543258/7604cfee-2294-11e6-8879-fe257f753125.png">